documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_3_App_Store_and_Data_Apps.md

```diff
@@ -1,43 +1,27 @@
 # Data Apps and App Store
```

Comment:
Also for this file: IDS App instead of Data Apps oder IDS Data Apps (unless of course the corresponding type is meant)